src/views/User/MyPostPage/MyPostPage.jsx

```diff
@@ -1,7 +1,57 @@
 import React from 'react';
+import styled from '@emotion/styled';
+import { ShortHeader } from '@/components/Header';
+import { PostCard } from '@/components/PostCard';
+import { postsData } from '@/assets/data.js';
 
 const MyPostPage = () => {
-  return <div></div>;
+  const { posts } = postsData;
+  const postLength = 0;
+  return (
+    <Wrapper>
+      <ShortHeader location="내가 쓴 글" />
+      <ContentWrapper>
+        {postLength ? (
+          <PostCardList>
+            {posts.map(({ id, ...props }) => (
+              <PostCardWrapper key={id}>
+                <PostCard {...props} />
+              </PostCardWrapper>
+            ))}
+          </PostCardList>
+        ) : (
+          <NoResultText>내가 쓴 글이 없습니다.</NoResultText>
+        )}
+      </ContentWrapper>
+    </Wrapper>
+  );
 };
 
 export default MyPostPage;
+
+const Wrapper = styled.div`
+  margin: 0 auto;
+`;
+
+const ContentWrapper = styled.div`
+  margin: 10rem 2.4rem 2.4rem 2.4rem;
+`;
+
+const PostCardList = styled.ul`
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1.2rem;
+  justify-content: space-between;
```

Comment:
```suggestion
  justify-content: center;
```

![image](https://user-images.githubusercontent.com/81365896/145941301-e19e143a-d530-4ba5-b951-64c79d668254.png)

![image](https://user-images.githubusercontent.com/81365896/145941356-572bdd04-c2f2-4d55-8651-cc3d153b0c3a.png)

space-between으로 하면 반응형에서 비어 있는 지점이 있네요

Reply:
게시글이 홀수일 때 이렇게 나와서 수정한 건데 center가 더 낫겠네요.
<img width="342" alt="image" src="https://user-images.githubusercontent.com/75232366/145949372-9d4bbbbf-06a6-4cf5-8d64-74f62b1e7e75.png">